cost/cloud_cost_anomaly_alerts/cloud_cost_anomaly_alerts.pt

```diff
@@ -192,45 +192,78 @@ script "js_filter_dimensions", type: "javascript" do
   parameters "param_dimensions", "ds_dimensions"
   result "filtered_dimensions"
   code <<-EOS
-    var filtered_dimensions = {
-      ids: [],
-      names: [],
-      invalid_ids: [],
-      filters:[]
-    };
-    var exist = false;
-
-    _.each(param_dimensions, function(user_input_dimension, index) {
-      exist = false;
-      filter = {
-        "dimensionid":"",
-        "dimensionname":"",
-        "type":"",
-        "value":""
-      }
-      dimension_filter_map = user_input_dimension.split("=")
-      hasfilter = false;
-      if (dimension_filter_map.length > 1){
-        hasfilter=true
+  var filtered_dimensions = {
```

Comment:
I think old indentation was right, with new one code can't collapse in code editor.